lib/ood_core/job/adapters/slurm.rb

```diff
@@ -699,6 +755,10 @@ def nodes
           @slurm.nodes
         end
 
+        def sacct_metrics(job_ids: [], states: [], from: nil, to: nil)
+          @slurm.sacct_metrics(job_ids, states, from, to)
+        end
```

Comment:
I think we should have an actual `historic_info` API on the adapter class itself to mimic and extend the `info` API.

Not 100% sure on the method signature here, but these are all keywords so it should be OK for now.

Note that this API should probably respond with an array of Info objects (currently returns an array of hashes?).

Reply:
I made some changes to return an array of info objects and added support to disable job steps.

Still when job steps are enabled, it will return them as regular info objects. Not sure this is the best approach at this point, but for our use case, we need the steps for memory metrics calculations.

Reply:
I think I also wanted the top level `historic_info` on the Adapter class that this is then the implementation for.

Reply:
OK - will make the changes and see how that looks.